src/day2.scala

```diff
@@ -0,0 +1,63 @@
+// using scala 3.0.2
+
+package day2
+
+import scala.io.Source
+
+@main def part1(): Unit =
+  val input = Source.fromFile("input/day2").mkString
```

Comment:
This is something I missed yesterday, but good practice is to `close()` the source after you have consumed it. I suggest using `Using`, which does this automatically for you. It would be something like:

```suggestion
  val input = util.Using.resource(Source.fromFile("input/day2"))(_.mkString)
```